translations/mod-fqm-manager/en.json

```diff
@@ -199,15 +199,15 @@
   "entityType.drv_fund_type_details.name": "Name",
   "entityType.drv_fund_type_details.version": "Version",
   "entityType.simple_group_details": "Group",
-  "entityType.simple_group_details.created_by_user_id": "Created by user ID",
+  "entityType.simple_group_details.created_by_user_id": "Created by user UUID",
   "entityType.simple_group_details.created_by_username": "Created by username",
   "entityType.simple_group_details.created_date": "Created date",
   "entityType.simple_group_details.desc": "Desc",
   "entityType.simple_group_details.expiration_off_set_in_days": "Expiration off set in days",
   "entityType.simple_group_details.group": "Group",
-  "entityType.simple_group_details.id": "ID",
+  "entityType.simple_group_details.id": "Group UUID",
```

Comment:
Should we be including "Group" here? E.g., in the users composite entity type, it will show up as "Group - Group UUID" - is that ok?